### PR TITLE
CommonGen data statement

### PR DESCRIPTION
**Pull request by yjernite**
This proposes a first version of a Dataset Sheet for the CommonGen Dataset.

cc @sebastianGehrmann

### CODE REVIEW

**Review by salomeyosei — APPROVED** [review]

**Comment by yjernite** [comment]
@sebastianGehrmann here's the updated version following the new template!

File: web/tasks/CommonGen.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    address = "Online",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/2020.findings-emnlp.165",
+    pages = "1823--1840", 
+}
+```
+
+### Leaderboard
+
+If the dataset supports an active leaderboard, add a link here and describe how models are evaluated.
```
Comment (sebastianGehrmann) [inline web/tasks/CommonGen.md:64]: Could you fill this?

File: web/tasks/CommonGen.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+### Data Fields
+
+A data instance has the following fields
+* `concepts`: a `list` of  `string` values denoting the concept the system should write about. Has 3 to 5 items, constitutes the `input` of the task.
+* `target`: a sentence `string` mentioning all of the above mentioned `concepts`. Constitutes the desired `output` of the task.
+
+### Data Statistics
+
+Each example in the dataset consists in a set of 3 to 5 concepts denoted by a single noun, verb, or adjective (the input), and a sentence using these concepts (the output). The dataset provides several such sentences for each such concept.
```
Comment (sebastianGehrmann) [inline web/tasks/CommonGen.md:95]: consists in -> consists of

File: web/tasks/CommonGen.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+A screenshot of the interface is provided in Figure 7 of the [Appendix](https://arxiv.org/pdf/1911.03705v3.pdf).
+
+#### Who are the source language producers?
+
+The original language of this dataset was created by Amazon Mechanical Turk workers.
+
+During the data collection, workers who provided rationales that were too short, failed to have good coverage of the input in their sentences, or workers whose output had a high perplexity under a GPT-2 model were disqualified from the pool and replaced with newcomers.
+
+No demographic information is provided
```
Comment (sebastianGehrmann) [inline web/tasks/CommonGen.md:152]: nit: end on period

File: web/tasks/CommonGen.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+### Annotations
+
+The dataset does not contain any additional annotations,
```
Comment (sebastianGehrmann) [inline web/tasks/CommonGen.md:156]: nit: period instead of comma

File: web/tasks/CommonGen.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+
+The dataset is created using data from image captioning systems and might inherit some of the social biases represented therein (see e.g. [Tang et al. 2020](https://arxiv.org/abs/2006.08315)).
+
+Another related concern is the exposure bias introduced by the initial selection of pictures and video, which are likely to over-represent situations that are common in the US at the expense of other parts of the world (Flickr, for example, is a US-based company founded in Canada). For more discussion of the potential impacts of exposure bias, see e.g. [The Social Impact of Natural Language Processing](https://www.aclweb.org/anthology/P16-2096.pdf).
+
+The use of GPT-2 to validate development ant test sentences [might be cause for similar concern](https://www.aclweb.org/anthology/D19-1339.pdf), but we do note that the authors only use the model to discount very high perplexity sequences which is less likely to surface those biases.
+
+### Other Known Limitations
+
+The language in the development and test set is crowdsourced, which means that it was written by workers whose main goal was speed. This is likely to impact the quality and variety of the targets.
```
Comment (sebastianGehrmann) [inline web/tasks/CommonGen.md:194]: Maybe also note that the population of turkers is not identically distributed as the overall population which may lead to different representation of situations or underlying expectations of what these situations are.

**Review by sebastianGehrmann — CHANGES_REQUESTED** [review]
Looking good to me, just a couple small comments.

**Pull request merged**